weaver/core/network/fabric-interop-cc/contracts/interop/manage_assets.go

```diff
@@ -144,7 +144,7 @@ func (s *SmartContract) UnlockAssetUsingContractId(ctx contractapi.TransactionCo
 	return nil
 }
 
-// ClaimAsset cc is used to record claim of an asset on the ledger (this uses the contractId)
+// ClaimAssetUsingContractId cc is used to record claim of an asset on the ledger (this uses the contractId)
```

Comment:
I think it's better to change the comment to `Record claim of an asset on the ledger (this uses the contractId)` as it's clearly describes the purpose and the name of the documented function is obvious (and less error prone to change in the future as we see now). Same applies to another change.